src/components/CommentList.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { ErrorType } from '../types/ErrorType';
+import { customFetch } from '../utils/fetchClient';
+import { Comment } from '../types/Comment';
+import { CommentInfo } from './CommentInfo';
+
+type Props = {
+  comments: Comment[];
+  setErrorType: React.Dispatch<React.SetStateAction<ErrorType | null>>;
+  setComments: React.Dispatch<React.SetStateAction<Comment[]>>;
+};
+
+export const CommentsList: React.FC<Props> = ({
```

Comment:
Props destructuring can be done directly in the function parameters for brevity and readability.